src/psbt.c

```diff
@@ -2726,7 +2726,7 @@ int wally_finalize_psbt(struct wally_psbt *psbt)
         }
 
         /* Note that if we patch libwally to supply the non-witness utxo tx field (tx) for
-         * witness inputs also, we'll need a different way to signal p2wpkh scripts */
+        * witness inputs also, we'll need a different way to signal p2sh-p2wpkh scripts */
```

Comment:
nit: indentation.

Reply:
turns out `uncrustify` wants it this way, so reverted this suggested change.